NBitcoin/Network.cs

```diff
@@ -252,6 +252,13 @@ public static Consensus TestNet
 				return Network.TestNet.Consensus;
 			}
 		}
+		public static Consensus TestNet4

```

Comment:
Remove.  Just use Bitcoin.Instance.Testnet4. testnet4 is a bitcoin-only concept.

Reply:
Done